src/main/js/README.md

```diff
@@ -0,0 +1,14 @@
+# Setting up Speech to Text Demo
+
+```sh
```

Comment:
I assume you are using Unity (default desktop environment for Ubuntu), which is probably not supported (yet).